app/lib/services/query_store.rb

```diff
@@ -39,8 +39,7 @@ def works_in_public_childcare_provider?
   end
 
   def works_in_private_childcare_provider?
-    works_in_nursery? &&
-      Forms::KindOfNursery::KIND_OF_NURSERY_PRIVATE_OPTIONS.include?(store["kind_of_nursery"])
+    works_in_childcare? && !works_in_public_childcare_provider?
```

Comment:
This is the core fix to CN-275. It makes it so that if you say you work in childcare and go down any path other than saying you work in a public nursery, you'll be treated like your work in a private childcare provider, which is how the questionnaire flow works.